tests/gordo_components/cli/test_cli.py

```diff
@@ -441,3 +442,33 @@ def test_server_to_sql_cli():
     args = ["workflow", "server-to-sql", "--help"]
     result = runner.invoke(cli.gordo, args)
     assert result.exit_code == 0
+
+
+def test_log_level_cli():
+    """
+    Test that the --log-level option in the CLI sets the correct log-level in the genreated config file.
+    """
+
+    runner = CliRunner()
+    args = [
+        "--log-level",
+        "TEST_LOG_LEVEL",
+        "workflow",
+        "generate",
+        "--machine-config",
+        "examples/config_crd.yaml",
+        "--project-name",
+        "test",
+    ]
+
+    result = runner.invoke(cli.gordo, args)
+
+    # Find the value on the next line after the key GORDO_LOG_LEVEL
+    gordo_log_levels = re.findall(
+        r"(?<=GORDO_LOG_LEVEL\r|GORDO_LOG_LEVEL\n)[^\r\n]+", result.stdout
+    )
+
+    # Assert all the values to the GORDO_LOG_LEVEL key contains the correct log-level
+    assert all(
+        [True if "TEST_LOG_LEVEL" in value else False for value in gordo_log_levels]
```

Comment:
_very_ minor, but you could have just done `"TEST_LOG_LEVEL" in value`

Reply:
`True - True` Will fix it